web/jobs/jobs.go

```diff
@@ -58,7 +61,39 @@ func pushJob(c echo.Context) error {
 		return wrapJobsError(err)
 	}
 
-	return jsonapi.Data(c, http.StatusAccepted, &apiJob{job}, nil)
+	accept := c.Request().Header.Get("Accept")
+	if accept != TypeTextEventStream {
+		return jsonapi.Data(c, http.StatusAccepted, &apiJob{job}, nil)
+	}
+
+	w := c.Response().Writer
+	w.Header().Set("Content-Type", TypeTextEventStream)
+	w.WriteHeader(200)
```

Comment:
Are you doing Server Sent Events on a POST request? I didn't know that it was possible to use that from a browser!